frontend/components/interests/form/InterestFormTextInput.tsx

```diff
@@ -9,6 +9,7 @@ interface TextInputProps {
   label: string;
   placeholder?: string;
   numeric?: boolean | undefined;
+  disableAutocomplete?: boolean | undefined;
```

Comment:
@zognin just realised you are working on these files as well for #130. Lets merge yours first. 

⚠️  Do not resolve this conversation, do not merge this PR until the above is done.